vision_datasets/image_object_detection/detection_as_kvp_dataset.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import typing
+from copy import deepcopy
+
+from vision_datasets.common import DatasetTypes, KeyValuePairDatasetInfo, VisionDataset
+from vision_datasets.key_value_pair import (
+    KeyValuePairDatasetManifest,
+    KeyValuePairLabelManifest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+BBOXES_KEY = "bboxes"
```

Comment:
Given how we are framing the task (grounding), I'd suggest replacing this key with `detectedObjects`, since `bboxes` is not as informative.

Reply:
agree, and also change BBOXES_KEY to something like OBJECTS_KEY